.github/dependabot.yml

```
@@ -0,0 +1,18 @@
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+  - package-ecosystem: npm
```

```suggestion
version: 2
updates:
  - package-ecosystem: npm
```

See: https://github.com/Pioneer-Valley-Books/postcss-dvh-vh-fallback/runs/21419201452

---

Oops. Deleted too many lines, will fix.

---

Fixed in https://github.com/Pioneer-Valley-Books/postcss-dvh-vh-fallback/pull/3